fe/src/main/java/org/apache/doris/load/loadv2/LoadManager.java

```diff
@@ -583,6 +595,27 @@ public void updateJobScannedRows(Long jobId, TUniqueId loadId, TUniqueId fragmen
         }
     }
 
+    // in previous implementation, there is a bug that when the job's corresponding transaction is
+    // COMMITTED but not VISIBLE, the load job's state is LOADING, so that the job may be CANCELLED
```

Comment:
I have a question, when job state is changed to cancelled, is the corresponding transaction state is checked now? If it isn't, I think it is better to add this check.